src/clojure/dialog/logger.clj

```diff
@@ -93,29 +93,37 @@
 
 (defn blocked?
   "True if the logger is blocked by configuration, otherwise false."
-  [logger]
-  (boolean
-    (and (string? logger)
-         (some #(prefixed? logger %)
-               (:blocked config)))))
+  ([logger]
+   (blocked? (:blocked config) logger))
+  ([levels logger]
+   (boolean
+     (and (string? logger)
+          (some #(prefixed? logger %)
+                levels)))))
 
 
-(defn- match-block
+(defn match-block
```

Comment:
I'm not seeing why this needs to be made public. Changing it to accept a set of prefixes to support the change below makes sense, though.

Reply:
Moved private. 
Might have been an overlook on my side.